src/inventory/gilded_rose.js

```diff
@@ -21,48 +21,49 @@ const items = [
 
 updateQuality(items);
 */
+
 export function updateQuality(items) {
-  for (var i = 0; i < items.length; i++) {
-    if (items[i].name != 'Aged Brie' && items[i].name != 'Backstage passes to a TAFKAL80ETC concert') {
-      if (items[i].quality > 0) {
-        if (items[i].name != 'Sulfuras, Hand of Ragnaros') {
-          items[i].quality = items[i].quality - 1
-        }
-      }
+  items.forEach((item) => {
+    const improvesWithAge = item.name === 'Aged Brie' || item.name === 'Backstage passes to a TAFKAL80ETC concert';
+    const constantQuality = item.name === 'Sulfuras, Hand of Ragnaros';
```

Comment:
It would be a good idea to put these hard coded values in constants incase they need to be used more than once.

```js
const AGED_BRIE = 'Aged Brie';
const BACKSTAGE_PASSES = 'Backstage passes to a TAFKAL80ETC concert';
const SULFURAS_HAND_OF_RAGNAROS =  'Sulfuras, Hand of Ragnaros';

// or

const ITEM_NAMES = {
  AGED_BRIE: 'Aged Brie',
  BACKSTAGE_PASSES: 'Backstage passes to a TAFKAL80ETC concert',
  SULFURAS_HAND_OF_RAGNAROS: 'Sulfuras, Hand of Ragnaros'
};
```